docs/writing-rules/private-rules.md

```diff
@@ -0,0 +1,34 @@
+---
+append_help_link: true
+slug: private-rules
+description: "Users in the Team and Enterprise tier for Semgrep App can publish rules to the Semgrep Registry that are not visible to others outside their organization. This can be useful for organizations where rules may contain code-sensitive information or legal requirements prevent using a public registry."
+---
+
+# Private rules
+
+Users in the [Team and Enterprise tier](https://semgrep.dev/pricing) for Semgrep App can publish rules to the Semgrep Registry that are not visible to others outside their organization. This can be useful for organizations where rules may contain code-sensitive information or legal requirements prevent using a public registry.
+
+As we continue to develop and refine this feature, we welcome and appreciate all feedback via email at product@returntocorp.com or in our [Community Slack](https://r2c.dev/slack).
+
+## Getting Started
+
+[View instructions](https://github.com/returntocorp/semgrep/tree/develop/upload) on how to beging uploading your private rules.
```

Comment:
I suggest putting the content in that `upload` file directly in the docs, that way it’s searchable and discoverable along with the rest of the docs.